firmware/rhizome.ino

```diff
@@ -76,20 +80,17 @@ void setup() {
 //    sprouts.push_back(new Ohmbrewer::HeatingElement( 2, new std::list<int>(1,5), 0, true, fakeTask ));
 
 // EX 4: A RIMS w/ a Pump to the Kettle
```

Comment:
Don't forget to update the other examples (1-3) above so that we know we haven't broken anything... I tried using Thermostat here and I get a hard fault, despite commenting things out in the setup() and loop()